src/common/services/tokenApi.js

```diff
@@ -29,15 +29,12 @@ module.exports = {
       }, (error, response, body) => {
         if (error) {
           logger.error('There was a problem calling the settoken API');
-          return console.dir(error);
+          logger.error(error);
+          reject(error);
```

Comment:
Potentially, if the endpoint ever rejected for any reason, then this Promise would be unhandled. There is no particular reason why this would not mean the application could have hung. Error is now wrapped into a reject call.

Other service calls may need looking at to ensure this is not the case.